.github/workflows/test_app.yml

```diff
@@ -0,0 +1,18 @@
+name: Testear la aplicación
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  testea_app:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Instalar task runner
+        run: pip install invoke
+      - name: Run tests
+        run: invoke rundocker
```

Comment:
Estás testeando lo mismo en los dos sistemas. No os digo que pongáis dos sistemas porque sí, sino para que testeéis cosas diferentes. Entre otras (y las más importantes) diferentes versiones del lenguaje.